pkgs/build-support/fetchpijul/nix-prefetch-pijul

```diff
@@ -125,10 +130,8 @@ EOF
 if [ -n "$change" ]; then cat <<EOF
 	"change": "$(json_escape "$change")",
 EOF
-elif [ -n "$state" ]; then cat <<EOF
-	"state": "$(json_escape "$state")",
-EOF
 fi; cat <<EOF
+	"state": $(json_escape "$state"),
```

Comment:
Yes, the straight quotation marks should be dropped here… see (& possibly review/merge plz): https://github.com/NixOS/nixpkgs/pull/464584